spicedb.go

```diff
@@ -122,34 +115,17 @@ func (customizer SecretKeyCustomizer) Customize(req *testcontainers.GenericConta
 }
 
 type ModelCustomizer struct {
-	Model     string
-	SecretKey string
+	Model         string
+	SecretKey     string
+	SchremaWriter func(ctx context.Context, c testcontainers.Container, model string, secret string) error
```

Comment:
If this is idiomatic to spicedb, then I'm fine with it. Another way would be to just demonstrate how to do this in a testable example. OTOH adding it here as part of the model struct makes it evident and explicit.

I'd say 👍